### DISCUSSION TITLE
How do i ship native modules with nexe

### DISCUSSION THREAD

**Discussion by bryiac** [nexe/nexe#864] [Q&A]
Been trying to ship native modules looked into nexe-natives and also tried including them as a resource but even if the .node file is in the right path it still complains is there any info on this

**Comment by hipstersmoothie** [comment]
This would be a killer feature that would really make me move away from pkg. It seem like it's almost possible. The module support in latest 14 is pretty nice and doesn't seem like it would be a huge lift to make happen
